src/main/java/com/hermesworld/ais/galapagos/adminjobs/impl/NoUpdatesAdminClient.java

```diff
@@ -191,54 +191,54 @@ public DescribeReplicaLogDirsResult describeReplicaLogDirs(Collection<TopicParti
 
     @Override
     public CreatePartitionsResult createPartitions(Map<String, NewPartitions> newPartitions) {
-        throw new UnsupportedOperationException();
+        return delegate.createPartitions(Map.of(), new CreatePartitionsOptions().validateOnly(true));
     }
 
     @Override
     public CreatePartitionsResult createPartitions(Map<String, NewPartitions> newPartitions,
             CreatePartitionsOptions options) {
-        throw new UnsupportedOperationException();
+        return delegate.createPartitions(Map.of(), new CreatePartitionsOptions().validateOnly(true));
```

Comment:
No need to pass empty partitions here. Just pass `newPartitions`, as it is still quite cool to have the server validate the set.